lightning/src/util/byte_utils.rs

```diff
@@ -69,11 +69,32 @@ pub fn be64_to_array(u: u64) -> [u8; 8] {
 	v[7] = ((u >> 8*0) & 0xff) as u8;
 	v
 }
+#[inline]
+pub fn be128_to_array(u: u128) -> [u8; 16] {
```

Comment:
ugh, we really need to increment our MSRV so we don't have to write this shit manually